deepchem/feat/molecule_featurizers/one_hot_featurizer.py

```diff
@@ -42,57 +44,121 @@ def __init__(self, charset: List[str] = ZINC_CHARSET, max_length: int = 100):
     self.charset = charset
     self.max_length = max_length
 
-  def _featurize(self, mol: RDKitMol) -> np.ndarray:
-    """Compute one-hot featurization of this molecule.
+  def featurize(self, datapoints: Iterable[Any],
+                log_every_n: int = 1000) -> np.ndarray:
+    """Featurize strings or mols.
 
     Parameters
     ----------
-    mol: rdkit.Chem.rdchem.Mol
-      RDKit Mol object
+    datapoints: A list of either strings or RDKit molecules.
+    log_every_n: int, optional (default 1000)
+      How many elements are featurized every time a featurization is logged.
+    """
+    datapoints = list(datapoints)
+    if (len(datapoints) < 1):
+      print(
```

Comment:
For code in the main DeepChem, you want to use the logging module rather than print statements. This statement can be removed since it's doing the default for an empty string and doesn't need to be logged